sci-electronics/minipro/minipro-0.5.ebuild

```diff
@@ -1,13 +1,14 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit bash-completion-r1 udev
```

Comment:
After these changes it looks like we no longer use `bash-completion-r1.eclass` so we can drop it from the inherit.

Reply:
This change is now in the ebuild.